src/atomizer.js

```diff
@@ -36,10 +36,14 @@ function Atomizer(options/*:AtomizerOptions*/, rules/*:AtomizerRules*/) {
 Atomizer.prototype.addRules = function(rules/*:AtomizerRules*/)/*:void*/ {
     rules.forEach(function (rule) {
         if (
-            (rule.type === 'pattern' && this.rulesMap.hasOwnProperty(rule.matcher)) ||
-            (rule.type === 'helper' && this.helpersMap.hasOwnProperty(rule.matcher))
+            (rule.type === 'pattern'
+                && this.rulesMap.hasOwnProperty(rule.matcher)
+                && !_.isEqual(this.rules[this.rulesMap[rule.matcher]], rule)) ||
```

Comment:
maybe we could add a getter on the prototype for `this.rules[this.rulesMap[rule.matcher]]` since it's operating on its own data and it would be self-documenting.